src/main/resources/application.conf

```diff
@@ -19,6 +19,9 @@ conseildb = {
 logger.scala.slick=INFO
 
 lorre: {
+  #Used to determine how many times Lorre should retry processing blocks/accounts
+  numberOfRetries: 10
```

Comment:
Might be a personal feeling, but this looks like a big number to me for a possibly-heavy operation.
What about 3 or 5?

Reply:
I would agree we should lower it to at least five.

Reply:
Sure thing!